examples/analog_in/voltage_acq_int_clk.py

```diff
@@ -0,0 +1,15 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to acquire a finite amount
+of data using the DAQ device's internal clock.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(1000.0, sample_mode=AcquisitionType.FINITE, samps_per_chan=1000)
+
+    data = task.read()
+    print(f"Acquired data: {data}")
```

Comment:
This example does a finite acquisition but it only reads and prints one sample. Use `task.read(READ_ALL_AVAILABLE)` to read all of the samples.

Once you do that, it's clear that 1000 is too many samples.

> PS D:\dev\nidaqmx-python> poetry run python .\examples\analog_in\voltage_acq_int_clk.py
Acquired data: [0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2724332141734568, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2724332141734568, 0.2722721419573308, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2724332141734568, 0.2722721419573308, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2724332141734568, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.2724332141734568, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.2724332141734568, 0.2722721419573308, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.2722721419573308, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27178892531054805, 0.27178892531054805, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.2724332141734568, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27178892531054805, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.2724332141734568, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2724332141734568, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.2724332141734568, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.2722721419573308, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.2716278530954854, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27178892531054805, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2724332141734568, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27178892531054805, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.2722721419573308, 0.2724332141734568, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.27211106974147065, 0.27178892531054805, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27194999752587645, 0.2724332141734568, 0.27194999752587645, 0.27194999752587645, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.2716278530954854, 0.27194999752587645, 0.2722721419573308, 0.27194999752587645, 0.2722721419573308, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27194999752587645, 0.27178892531054805, 0.27178892531054805, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27211106974147065, 0.27194999752587645, 0.2722721419573308, 0.2722721419573308, 0.2722721419573308, 0.27194999752587645]

Reply:
1000 samples are too many, should I reduce `samps_per_chan` to, maybe 30 or less?

Reply:
50 fits on the screen, which is a start.

![image](https://github.com/ni/nidaqmx-python/assets/17914399/099bccc9-0d44-4542-b55d-e303c397a304)

Reply:
Added `pprint` to print data, all 50 sample data fit in the screen.

![image](https://github.com/ni/nidaqmx-python/assets/27721199/9c16eca8-e09d-4b32-9b99-80f82982c738)

Reply:
They still have excess precision. Please use a formatting scheme that supports `{value:f}` or `%f`

Reply:
Okay, it's updated as per suggestion.